core/shared/src/main/scala/org/http4s/CacheDirective.scala

```diff
@@ -1,11 +1,17 @@
 /*
- * Copyright 2013-2020 http4s.org
+ * Copyright 2013 http4s.org
  *
- * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
  *
- * Based on https://github.com/spray/spray/blob/v1.1-M7/spray-http/src/main/scala/spray/http/CacheDirective.scala
- * Copyright (C) 2011-2012 spray.io
- * Based on code copyright (C) 2010-2011 by the BlueEyes Web Framework Team
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```

Comment:
We lost a credit here (and in a few other places).  There's a list of files not to touch with generated headers in Http4sPlugin.scala.

Reply:
Yes, I know, sorry about it :( it got really messed up with the directory shuffling, despite my attempt to adjust. I can try again. Is there a way we can just have both headers, so the auto-generator can be happy?

Reply:
I never figured out a way.  sbt-header wants to control everything above the package line.  I guess attribution could appear somewhere below, like on the types.  Honestly, most of these files have changed enough that they barely resemble the originals, but I still try hard to respect original authors and their licenses.  We're not Microsoft.

My solution was to enumerate the files not to touch in a list in the build, which is fine until files start to move.

Reply:
I think I fixed all of these. `sbt-header` seems to be a little more friendly nowadays, as long as its on top.